doc/api/stream.md

```diff
@@ -309,7 +309,7 @@ a readable stream, adding this writable to its set of destinations.
 const writer = getWritableStreamSomehow();
 const reader = getReadableStreamSomehow();
 writer.on('pipe', (src) => {
-  console.error('something is piping into the writer');
+  console.log('something is piping into the writer');
```

Comment:
While we're in here, can you change the text to capitalize the first letter and add a period afterwards so as to be consistent with the other examples?

Reply:
Good spotting @Trott.
Done.